### DISCUSSION TITLE
Seperate repo for protocol (SDK proto files)

### DISCUSSION THREAD

**Discussion by jibon57** [8xFF/atm0s-media-server#361] [Ideas]
Hello again,

At present SDK proto files are located inside the main atm0s-media-server repo e.g. https://github.com/8xFF/atm0s-media-server/tree/master/packages/protocol/proto/sdk. I think it's better to separate it & use separate repo. This way user can easily use/import those into their application in different language for example: Go, PHP etc. In atm0s-media-server repo you may import as submodule.

Thanks

**Comment by giangndm** [comment]
@jibon57 It's a good idea. The main reason why I chosen all in one repo is because it easier in early development.

May be we can refer to livekit protocol repo: https://github.com/livekit/protocol/
